src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        File fromFile = new File(fromFileName);
+        int supplyCount = 0;
+        int buyCount = 0;
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fromFile));
```

Comment:
creating File for that is not neccesary
```suggestion
        int supplyCount = 0;
        int buyCount = 0;
        try {
            BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
```